comit/src/float_math.rs

```diff
@@ -0,0 +1,223 @@
+use bitcoin::hashes::core::cmp::Ordering;
```

Comment:
So actually the name of the file is incorrect 😁 We are doing fixed point arithmetic here.
The aim of the module is to brought some safety in handling fractional numbers and doing maths/arithmetic with them. Some ideas for a better name:
- fraction
- fractional number
- maths
- arithmetic
- fixed point

Reply:
+1 for `fraction`